salt/_modules/caasp_net.py

```diff
@@ -23,10 +23,11 @@ def get_hosts():
     :return: Dict(ip_address -> [aliases])
     """
     hosts = __salt__['hosts.list_hosts']()  # type: dict
+    copy = hosts.copy()
```

Comment:
`hosts_copy`?

Reply:
can we add a comment here above : 
`In python3 python prevents modifying the dictionary that is iterated over.`